### PR TITLE
docs: capitalize JavaScript and TypeScript properly

### PR DESCRIPTION
**Pull request by ericcrosson-bitgo**
As seen here[^1] and here[^2].

[^1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript
[^2]: https://www.typescriptlang.org/

### CODE REVIEW

File: modules/statics/README.md [hunk 1]
```diff
@@ -15,15 +15,15 @@
 
 ### Get the number of decimal places in a Bitcoin
 
-#### Javascript
+####javascript
```
Comment (tylerlevine) [inline modules/statics/README.md:18]: These look like broken section headers to me now when I preview this file.

Comment (tylerlevine) [reply modules/statics/README.md:18]: image

**Review by tylerlevine — CHANGES_REQUESTED** [review]
looks good, just need to fix the section headers. Thanks!

File: modules/statics/README.md (hunk 1 above)
Comment (ericcrosson-bitgo) [reply modules/statics/README.md:18]: Whoopsie

My brain's markdown parser malfunctioned, I thought that was a code block. Un momento

Comment (ericcrosson-bitgo) [reply modules/statics/README.md]: Fixed 3 occurrences

**Comment by ericcrosson-bitgo** [comment]
image

**Review by 0xabrarhussain — APPROVED** [review]

File: modules/statics/README.md [hunk 2]
```diff
@@ -33,15 +33,15 @@ console.log(btc.decimalPlaces);
...
-#### Javascript
+#### JavaScript
 ```js
 const { coins } = require('@bitgo/statics');
 
 const omg = coins.get('omg');
 console.log(omg.contractAddress);
 ```
 
-#### Typescript
```
Comment (0xabrarhussain) [inline modules/statics/README.md:44]: actually missed this. was merged but think this is a typo. same with other places it got changed back to `typescript`

**Pull request merged**